### DISCUSSION TITLE
Add Show/Hide Validation Errors option for Disabled Nodes/Flows

### DISCUSSION THREAD

**Discussion by Balok-Fesarius** [zachowj/node-red-contrib-home-assistant-websocket#1518] [Feature Request]
Often I will throw sample flows I’m interested in on tabs and disable them and keep them for reference.  Would be nice to be able to not see all the nodes/flows validation issues for stuff I never plan to deploy.  ¯\_(ツ)_/¯

**Comment by zachowj** [comment]
What version are you running? By default disabled nodes should not show.

**Reply by Balok-Fesarius** [reply]
0.68.7...   What about when individual nodes aren't disabled, but the entire flow (tab) is?

**Reply by Balok-Fesarius** [reply]
Looks like you fixed it in 0.68.9! Thanks.
